pkg/ddc/alluxio/shutdown.go

```diff
@@ -213,8 +207,6 @@ func (e *AlluxioEngine) destroyWorkers(workers int32) (err error) {
 				return err
 			}
 		}
-
-		i++
 	}
```

Comment:
Could we also print the node list which is going to remove label? It's useful for debugging.

Reply:
Yep. It is necessary.